### PR TITLE
Update Housing Subsidy Administrators Translation to be Route Specific

### PR DESCRIPTION
**Pull request by dtgreiner**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

Update Match Route 6 to have route specific translations.

## Type of change
[//]: # 'remove options that are not relevant'
- [X] New feature (adds functionality)

## Checklist before requesting review
- [X] I have performed a self-review of my code
- [X] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [X] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [X] My code follows the style guidelines of this project (rubocop)
- [X] I have updated the documentation (or not applicable)
- [ ] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: app/views/notifications_mailer/six_match_recommendation_shelter_agency.text.haml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Hello #{@contact.full_name},
+\
+CAS has a new housing recommendation that requires your review.
+\
+Your client has matched to a vacancy in a housing navigation program. Please connect with housing navigation program staff to discuss this opportunity with the client. You should not accept or decline the opportunity in the CAS system prior to the client discussing the opportunity with housing navigation program staff:
```
Comment (eanders) [inline app/views/notifications_mailer/six_match_recommendation_shelter_agency.text.haml]: @dtgreiner let's restore the original text (or maybe something similar) and implement a translation of this paragraph.  That way the text can be adjusted as necessary.

File: app/models/match_decisions/six/approve_match_housing_subsidy_admin.rb [hunk 2]
```diff
@@ -15,9 +15,9 @@ def label
 
     def label_for_status status
       case status.to_sym
-      when :pending then "#{Translation.translate('Housing Subsidy Administrator')} reviewing match"
-      when :accepted then "Match accepted by #{Translation.translate('Housing Subsidy Administrator')}"
-      when :declined then "Match declined by #{Translation.translate('Housing Subsidy Administrator')}.  Reason: #{decline_reason_name}"
+      when :pending then "#{Translation.translate('Housing Subsidy Administrator Six')} reviewing match"
+      when :accepted then "Match accepted by #{Translation.translate('Housing Subsidy Administrator Six')}"
+      when :declined then "Match declined by #{Translation.translate('Housing Subsidy Administrator Six')}.  Reason: #{decline_reason_name}"
```
Comment (eanders) [inline app/models/match_decisions/six/approve_match_housing_subsidy_admin.rb:20]: I think we should add a migration to set the defaults for all of these `Six` specific translations to be translated as the previous text.  So for example `CoC Staff` would be translated as `DND` and `Housing Subsidy Administrator Six` as `Housing Subsidy Administrator`.  That way if anyone _is_ using the route, the names won't change, but we can translate as necessary.

**Review by eanders — CHANGES_REQUESTED** [review]
Thanks @dtgreiner this looks great, just a couple of tweaks please!

File: app/views/notifications_mailer/six_match_recommendation_shelter_agency.text.haml (hunk 1 above)
Comment (dtgreiner) [reply app/views/notifications_mailer/six_match_recommendation_shelter_agency.text.haml]: @eanders This has been addressed in the PR. The language was being used elsewhere, so I appended 'Six' to the end similar to how the rest of the new translations were created.

File: app/models/match_decisions/six/approve_match_housing_subsidy_admin.rb (hunk 2 above)
Comment (dtgreiner) [reply app/models/match_decisions/six/approve_match_housing_subsidy_admin.rb:20]: @eanders This has been added to the PR

File: app/views/notifications_mailer/six_match_recommendation_shelter_agency.text.haml [hunk 3]
```diff
@@ -0,0 +1,17 @@
+Hello #{@contact.full_name},
+\
+CAS has a new housing recommendation that requires your review.
+\
+#{Translation.translate('View details here: Six')}
```
Comment (eanders) [inline app/views/notifications_mailer/six_match_recommendation_shelter_agency.text.haml:5]: Thanks!

File: app/models/match_routes/six.rb [hunk 4]
```diff
@@ -59,5 +59,18 @@ def status_declined?(match)
...
+      case contact_type
+      when :dnd_staff_contacts
+        Translation.translate('CoC Six')
+      when :housing_subsidy_admin_contacts
+        Translation.translate('Housing Subsidy Administrator Six')
+      when :shelter_agency_contacts
+        Translation.translate('Shelter Agency Six')
+      else
+        super(contact_type)
+      end
```
Comment (eanders) [inline app/models/match_routes/six.rb:73]: Thanks!  That was it!

**Review by eanders — APPROVED** [review]

**Pull request merged**